presentation/src/main/java/com/example/dms_android/feature/navigator/DmsApp.kt

```diff
@@ -0,0 +1,162 @@
+package com.example.dms_android.feature.navigator
+
+import android.annotation.SuppressLint
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.shape.CornerSize
+import androidx.compose.material.BottomAppBar
+import androidx.compose.material.BottomNavigationItem
+import androidx.compose.material.Icon
+import androidx.compose.material.MaterialTheme
+import androidx.compose.material.Scaffold
+import androidx.compose.material.rememberScaffoldState
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.saveable.rememberSaveable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.unit.dp
+import androidx.navigation.NavGraph.Companion.findStartDestination
+import androidx.navigation.NavHostController
+import androidx.navigation.compose.NavHost
+import androidx.navigation.compose.composable
+import androidx.navigation.compose.rememberNavController
+import com.example.design_system.color.DormColor
+import com.example.dms_android.feature.cafeteria.CafeteriaScreen
+import com.example.dms_android.feature.mypage.MyPageScreen
+import com.example.dms_android.feature.notice.NoticeScreen
+
+@SuppressLint("UnusedMaterialScaffoldPaddingParameter")
+@Composable
+fun DmsApp() {
+    val navController = rememberNavController()
+    val scaffoldState = rememberScaffoldState()
+    val navHostController = rememberNavController()
+
+    Scaffold(
+        scaffoldState = scaffoldState,
+        bottomBar = { BottomNavBar(navController = navHostController) },
+        modifier = Modifier
+            .background(DormColor.Gray900),
+    ) { innerPadding ->
+        NavHost(
+            navController = navHostController,
+            startDestination = BottomNavigationItem.Meal.route,
+            Modifier.padding(innerPadding)
```

Comment:
```suggestion
            modifier = Modifier.padding(innerPadding)
```
nameArgument가 생략되었어요.